src/ui.rs

```diff
@@ -37,17 +37,15 @@ fn progress_indicator_setting(config: &config::Config) -> bool {
 }
 
 #[derive(Clone, Copy, Debug, Eq, PartialEq)]
+#[derive(Default)]
```

Comment:
nit:
```suggestion
#[derive(Clone, Copy, Debug, Eq, PartialEq, Default)]
```
(and below)

Reply:
Fixed. (That change was made by clippy but I should have noticed.)